CHANGELOG.md

```
@@ -1,3 +1,8 @@
+## 0.0.7
+
+* fix tests 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)